build.gradle

```diff
@@ -167,6 +167,9 @@ subprojects {
             destinationFile = file("$buildDir/jacoco/${name}.exec")
             excludes = ['**/Immutable*', '*Immutable*', 'generated/**']
         }
+
+        // for BqDdlOperatorFactoryTest
+        environment "GOOGLE_APPLICATION_CREDENTIALS", System.getProperty("java.io.tmpdir") + "/dummy-service-account-key.json"
```

Comment:
Is there any reason why you are passing this via an environment variable? If not, how about putting it in GcpCredentialProviderTest.java?